java-client/src/main/java/org/opensearch/client/opensearch/_types/StoreStats.java

```diff
@@ -57,18 +57,18 @@ public class StoreStats implements JsonpSerializable {
 	@Nullable
 	private final String size;
 
-	private final int sizeInBytes;
+	private final long sizeInBytes;
 
 	@Nullable
 	private final String reserved;
 
-	private final int reservedInBytes;
+	private final long reservedInBytes;
 
 	@Nullable
 	private final String totalDataSetSize;
 
 	@Nullable
-	private final Integer totalDataSetSizeInBytes;
+	private final Long totalDataSetSizeInBytes;
```

Comment:
I don't see this field in the response from API in version 2.7.0
Seems like it was never produced by the server:
https://github.com/opensearch-project/OpenSearch/blob/1.0/server/src/main/java/org/opensearch/index/store/StoreStats.java

Should I remove it or mark it as @Deprecated?

Reply:
Mark it as deprecated, will remove in the next major version.

Reply:
Done in: https://github.com/opensearch-project/opensearch-java/issues/497 :)